test/bundle.js

```diff
@@ -35,30 +35,22 @@ mocha.describe('watching', function () {
     fs.writeFileSync(file2.name, makeRequire(file3.name));
     fs.writeFileSync(file3.name, makeRequire(file4.name));
 
-    bundle = galv.bundle({
-      paths: [
-        file1.name,
-        file2.name
-      ],
-      options: {
-        common: file1.name
-      }
-    });
```

Comment:
Yeah that was changed in ubercod and not updated here because the tests were already broken and I update separately anyways, as embarrassing as that is to admit. Thanks, Nic!